README.md

```diff
@@ -3,58 +3,74 @@
 This project provides common APIs for applications that live on vets.gov.
 
 ## Developer Setup
-Vets-api requires:
-- postgres
+
+Vets API requires:
+- PostgreSQL
 - Redis
-- rails server
+- Ruby 2.3
 
 ### Base Setup
 
-To start, fetch this code: `git clone https://github.com/department-of-veterans-affairs/vets-api.git`
+To start, fetch this code:
+
+`git clone https://github.com/department-of-veterans-affairs/vets-api.git`
 
-#### Automated
+#### Automated (OSX)
 
-*From the `vets-api` directory, run `./bin/setup-osx && source ~/.bash_profile && cd .` if you're on a mac. It will ensure that you have all development dependencies setup*
+If you are developing on OSX, you can run the automated setup script. From
+the `vets-api` directory, run `./bin/setup-osx && source ~/.bash_profile && cd .`
 
-#### Alternative
-1. Install Ruby 2.3. (It is suggested to use a Ruby version manager such as [rbenv](https://github.com/rbenv/rbenv#installation) and then to [install Ruby 2.3](https://github.com/rbenv/rbenv#installing-ruby-versions)).
-*Note*: rbenv will also provide additional installation instructions in the console output. Make sure to follow those too.
-1. Install Bundler to manage dependencies: `gem install bundler`
-1. Install Postgres (on Mac): `brew install postgres`
+#### Alternative (OSX)
 
-1. Install Redis (on Mac): `brew install redis`
+1. Install Ruby 2.3.
+  - It is suggested that you use a Ruby version manager such as
+    [rbenv](https://github.com/rbenv/rbenv#installation) and
+    [install Ruby 2.3](https://github.com/rbenv/rbenv#installing-ruby-versions).
+  - *Note*: rbenv will also provide additional installation instructions in the
+    console output. Make sure to follow those too.
+1. Install Bundler to manage dependencies
+  - `gem install bundler`
+1. Install Postgres and enable on startup
+  - `brew install postgres`
+  - `brew services start postgres`
+1. Install Redis
+  - `brew install redis`
+  - Follow post-install instructions to enable Redis on startup. Otherwise,
+    launch it manually with `brew services start redis`.
 1. Install gem dependencies: `cd vets-api; bundle install`
 1. Install overcommit `overcommit --install --sign`
-1. Create a application.yml `cat ./config/application.yml.example > ./config/application.yml`
 1. Setup localhost certificates / keys
-  - Create a hidden folder in home directory:  `mkdir ~/.certs`
-  - Copy the [certificate](https://github.com/department-of-veterans-affairs/vets.gov-team/blob/master/Products/Identity/Identity%20Discovery%202016/certificates/vetsgov-localhost.crt) to ~/.certs
-  - Copy the [key](https://github.com/department-of-veterans-affairs/vets.gov-team/blob/master/Products/Identity/Identity%20Discovery%202016/certificates/vetsgov-localhost.key) to ~/.certs
+  - Create a hidden folder in your home directory:  `mkdir ~/.certs`
+  - TODO(knkski): These are in a private repo...
```

Comment:
@aub: These certs are now residing in a private repo. Do we want to make those available somehow, or just add a note telling external contributors to do something like `touch ~/.certs/vetsgov-localhost.crt`?